styles/find-and-replace.less

```diff
@@ -374,3 +374,7 @@ atom-workspace.find-visible {
     }
   }
 }
+
+.list-tree.has-collapsable-children .list-nested-item > .list-item::before {
+  vertical-align: middle;
```

Comment:
It's related to that arrow icon showing that you can expand a nested item. We want them to keep it in the middle (vertically) at all times when you change the font size.